bin/serverless.js

```diff
@@ -14,7 +14,7 @@ const nodeVersion = Number(process.version.split('.')[0].slice(1));
 if (nodeVersion < 12) {
   const serverlessVersion = Number(require('../package.json').version.split('.')[0]);
   process.stdout.write(
-    `Serverless: \x1b[91mInitialization error: Node.js v${nodeVersion} is not supported by ` +
+    `\x1b[91mInitialization error: Node.js v${nodeVersion} is not supported by ` +
       `Serverless Framework v${serverlessVersion}. Please upgrade\x1b[39m\n`
```

Comment:
Could we mention the minimum required version?

Because so far the user doesn't know to which version to upgrade?

Reply:
In all cases, ideally, if users simply upgrade to latest LTS version of Node.js

So what final message do you propose?

Reply:
Something like:

> Please upgrade to version XX or greater.

Reply:
So your proposal is:

```
Error: Node.js v10 is not supported by Serverless Framework v3 Please upgrade to version v12 or greater
```
?

If that's the case, it feels to me as we'd suggest to install  v12 in first place (and that one becomes EOL in 3 months).

How about:

```
Error: Node.js v10 is not supported by Serverless Framework v3. Please upgrade Node.js to the latest LTS version (v12 is a minimum supported version)
```

?

Reply:
👍